content/docs/1.9.0/important-notes/_index.md

```diff
@@ -78,6 +78,7 @@ Automated checks are only performed on some upgrade paths, and the pre-upgrade c
 - Ensure that all V2 Data Engine volumes are detached and the replicas are stopped.  The V2 Data Engine currently does not support live upgrades.
 - Avoid upgrading when volumes are in the "Faulted" status.  If all the replicas are deemed unusable, they may be deleted and data may be permanently lost (if no usable backups exist).
 - Avoid upgrading if a failed BackingImage exists.  For more information, see [Backing Image](../advanced-resources/backing-image/backing-image).
+- It is recommended to create a system backup before performing the upgrade. This ensures that all critical resources, such as volumes and backing images, are backed up and can be restored in case any issues arise.
```

Comment:
Users might not know that `system backup` refers to Longhorn system backup. Consider using the term `Longhorn system backup` and link the term to the Longhorn system backup page for clarity.

Reply:
Fixed, thanks